src/SourceBuild/tarball/patches/aspnetcore/0001-Source-build-should-also-use-the-release-runtime.patch

```diff
@@ -0,0 +1,28 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Chris Rummel <crummel@microsoft.com>
+Date: Fri, 27 Jan 2023 14:34:40 -0600
+Subject: [PATCH] Source-build should also use the release runtime.
+
```

Comment:
```suggestion

Backport: https://github.com/dotnet/aspnetcore/pull/46299.
```

Reply:
I see the PR is marked as draft.  Ideally the PR would be approved prior to merging this PR.